build/controllers/FixLinksController.php

```diff
@@ -0,0 +1,167 @@
+<?php
+/**
+ * @link http://www.yiiframework.com/
+ * @copyright Copyright (c) 2008 Yii Software LLC
+ * @license http://www.yiiframework.com/license/
+ */
+
+namespace yii\build\controllers;
+
+use yii\build\helpers\LinkChecker;
+use yii\console\Controller;
+use yii\console\ExitCode;
+use yii\helpers\FileHelper;
+use Yii;
+
+/**
+ * Replaces outdated links(3xx response codes) add collect removed links (4xx response codes) in file
+ * `./build/runtime/outdated-links.log`.
```

Comment:
I don't think collecting links in a file is useful as GitHub action. It should be stdout in order to be visible in results at GitHub.

Reply:
@samdark no problem, I was logging removed links because this workflow running by Cron